examples/esm-content-script-ui/entrypoints/content/esm-index.ts

```diff
@@ -1,11 +1,15 @@
-import { ContentScriptContext } from "wxt/client";
+// This file is built using separate vite config from other WXT entrypoints, so
+// it can't import from `#imports`.
+import { ContentScriptContext } from "wxt/utils/content-script-context";
 import "./styles.css";
 
 export default async (ctx: ContentScriptContext) => {
   // Just demoing that dynamic imports work and support chunking, see
   // examples/esm-content-script-setup/.output/chrome-mv3/content-scripts/esm
   // This could be a normal import
-  const { createShadowRootUi } = await import("wxt/client");
+  const { createShadowRootUi } = await import(
+    "wxt/utils/content-script-ui/shadow-root"
+  );
```

Comment:
Hmm, I'd prefer to use the built-in vite config so all this resolves correctly... I wonder if instead of doing a separate vite build here, I could just add the file to the entrypoints HTML file group, which has chunking enabled?